.bingo/variables.env

```diff
@@ -8,11 +8,17 @@ if [ -z "$GOBIN" ]; then
 fi
 
 
-GOJSONTOYAML="$GOBIN/gojsontoyaml-v0.0.0-20200602132005-3697ded27e8c"
+BINGO="${GOBIN}/bingo-v0.3.0"
 
-JB="$GOBIN/jb-v0.4.0"
+CONTROLLER_GEN="${GOBIN}/controller-gen-v0.4.1"
 
-JSONNET="$GOBIN/jsonnet-v0.16.0"
+GOJSONTOYAML="${GOBIN}/gojsontoyaml-v0.0.0-20200602132005-3697ded27e8c"
 
-JSONNETFMT="$GOBIN/jsonnetfmt-v0.16.0"
+JB="${GOBIN}/jb-v0.4.0"
+
+JSONNET="${GOBIN}/jsonnet-v0.16.0"
```

Comment:
note: there is 0.17.0 with some performance improvements ;)

Reply:
Yup, let's not overcrowd this PR though (: